wordpress-mobile/WordPress-iOS
# Fix how the app handles edits made to autosaved drafts.

Fixes #10248.

This has more comments than code changes itself, but I believe this behaviour is edge-case'y enough to necessitate it. Hopefully the comments make clear what's going on, but if not — feel free to ping me!

(I'd appreciate a review from both of you @pinarol and @diegoreymendez — I'm honestly a bit scared of touching this logic, because of potential data loss but I've tested this enough that I'm _fairly sure_ it's fine. Another pair of eyes definitely won't hurt though!) 

To test:
Follow steps in #10248 and verify that the problem no longer exists.
Verify that the edited post still has a "Local" badge next to it displayed in the app.

Update release notes:
